scripts/plot_weather.py

```diff
@@ -701,7 +701,13 @@ def save_plot(self, plot_filename=None):
             os.makedirs(plot_dir)
 
         print('Saving Figure: {}'.format(plot_filename))
-        self.fig.savefig(plot_filename, dpi=self.dpi, bbox_inches='tight', pad_inches=0.10)
+        self.fig.savefig(
```

Comment:
Could you add a link to the discussion online about bbox_extra_artists?

Reply:
Done, thanks.